src/main/java/tdl/s3/sync/Destination.java

```diff
@@ -41,10 +57,67 @@ public AWSSecretsProvider getSecret() {
         return secret;
     }
 
-    public AmazonS3 buildClient() {
-        return AmazonS3ClientBuilder.standard()
+    public AmazonS3 getClient() {
```

Comment:
`UnfinishedWritingFileUploadingStrategyTest` can be removed for now. It is preventing us from changing the code. We will focus on the code coverage metric in a future iteration.

`getAlreadyUploadedParts` and `getPartListing` can move inside `Destination`